tutorials/vue/vue-d3.md

```diff
@@ -0,0 +1,151 @@
+[D3.js](https://d3js.org/) is a great library for visualizing data and displaying it in your projects.
+This library gives you the building blocks to make cool charts in projects.
+
+Here is an example of using `D3.js` in Vue2 to create a bar graph.
+Below is the code:
+
+```html
+<script src="https://d3js.org/d3.v6.js"></script>
+<script src="https://unpkg.com/vue"></script>
+<div class = "app">{{message}}
+<bargraph></bargraph>
+</div>
+<script type = "text/javascript">
+Vue.component('bargraph', {
+    mounted(){
+        this.retrieveGraph();
+    },
+    methods: {
+        retrieveGraph: function(){
+            let margin = ({top: 30, right: 30, bottom: 70, left: 60});
+            let width = 460 - margin.left - margin.right;
+            let height = 400 - margin.top - margin.bottom;
+            let info = [{Country: "United States", Value: "12394"},
+            {Country: "Russia",Value: "6148"},
+            {Country: "Germany (FRG)",Value: "1653"},
+            {Country: "France",Value: "2162"},
+            {Country: "United Kingdom",Value: "1214"},
+            {Country: "China",Value: "1131"},
+            {Country: "Spain",Value: "814"},
+            {Country: "Netherlands",Value: "1167"},
+            {Country: "Italy",Value: "660"},
+            {Country: "Israel",Value: "1263"}];
+            var svg = d3.select(this.$refs.example)
+                .append("svg")
+                .attr("width", width + margin.left + margin.right)
+                .attr("height", height + margin.top + margin.bottom)
+                .append("g")
+                .attr("transform",
+                "translate(" + margin.left + "," + margin.top + ")");
```

Comment:
Move this up a line, `.attr()` is one function call so this `translate` should be on the same line. Also, use a template string here for [string interpolation](https://masteringjs.io/tutorials/fundamentals/string-interpolation)